notebooks/01_structured.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<p>could you add a docstring to the function?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>done!</p>